cishouseholds/pipeline/bloods_delta_ETL.py

```diff
@@ -0,0 +1,36 @@
+from cishouseholds.derive import assign_column_uniform_value
+from cishouseholds.derive import substring_column
+
+
+def bloods_delta_ETL():
+    extract_bloods_delta()
+    transform_bloods_delta()
+    load_bloods_delta()
+
+
+def extract_bloods_delta():
+    pass
+
+
+def transform_bloods_delta(df):
+    """
+    Call functions to process input for bloods deltas.
+    D1: substring_column(new_column_name: 'plate', column_to_substr: 'plate_tdi', start_position: 5, len_of_substr: 10)
```

Comment:
The value under the last argument here doesn't match the one below, but I think the one used below is correct?

Also, it might be worth removing the arguments used from the docstring here, as it duplicates the code below. We could shorten these to just the logic id and function name? `D1: subtring_column`

Reply:
Removed arguments from doc strings